actions/utils/context.ts

```diff
@@ -203,3 +207,19 @@ function handlePromiseErrors<T>(
       return false;
     });
 }
+
+/**
+ * Init Logging with Loggly
+ */
+async function _initLogging(
+  transactionName: string,
+  chainId: SupportedChainId,
+  context: Context
+) {
+  const logglyToken = await context.secrets.get("LOGGLY_TOKEN").catch(() => "");
+  if (logglyToken) {
+    initLogging(logglyToken, [transactionName, `chain_${chainId}`]);
+  } else {
+    console.warn("LOGGLY_TOKEN is not set, logging to console only");
+  }
```

Comment:
Since this fails silently, how can we tell when loggly is turned on in production?

Reply:
We can see it in Tenderly log, or in local. 
I don't care too much, we can see it by logs not being pushed to Loggly